json_merger/dict_merger.py

```diff
@@ -25,7 +25,7 @@
 from __future__ import absolute_import, print_function
 
 import copy
-from itertools import chain
+import logging
```

Comment:
if we started using logging now, it would be better to use structlog instead, wdyt?

Reply:
I'm not against it, but we don't use `structlog` in any libraries, and the merger gets used by `next` which doesn't use `structlog` currently. Plus the logging here is only for debugging interactively, so `structlog` doesn't buy you much. So my opinion is that it's probably not worth it in this specific case.

Reply:
Indeed!